roles/fairfood/templates/post-receive.j2

```diff
@@ -53,6 +56,13 @@ upgrade_application() {
   echo "Restart puma..."
   bundle exec pumactl restart # Hot restart helps reduce downtime
 
+  echo "Restarting job workers..."
+  systemctl --user restart "jobs-{{ app }}.service"
+
+  # Ansible fails to interact with user services.
+  # But it's super-fast and idempotent. So we do it here.
+  systemctl --user enable jobs-fairfood.service
```

Comment:
I'm wondering if we should execute a general setup script during deploy. We already install Ruby and Bundler if needed. We could actually keep the service file template in the fairfood repository and copy it ourselves as well. Same with puma. We would reduce the circular dependencies between fairfood and fairfood-install. But for now, Ansible is just so convenient because it does the filling out of the template with existing variables for us. Well, one step at a time, let's try this approach of user-owned systemd files first before making more changes.

Reply:
Seems a bit strange to do one-time setup here, but hey like you said there's no real downside. And it helps ensure that it's enabled!

The only concern I can think of is during a migration to another server, we might want to keep it disabled. But we'd need to be commenting out parts of this script anyway, so we'd just need to comment this line too.